src/nidcpower/metadata/attributes.py

```diff
@@ -666,7 +666,7 @@
         'name': 'MEASURE_COMPLETE_EVENT_DELAY',
         'resettable': False,
         'type': 'ViReal64',
-        'type_in_documentation': 'float in seconds or datetime.timedelta'
+        'type_in_documentation': 'float in seconds or hightime.timedelta'
```

Comment:
It is regrettable that we have this documentation copied over and over and over.
You'd hope we could derive it from the `atttribute_class`.

Reply:
Agreed. Created an issue for it - https://github.com/ni/nimi-python/issues/1455.